security-setup

```diff
@@ -1,6 +1,15 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
This was changed intentionally to `python2` to comply with [PEP-0394](https://www.python.org/dev/peps/pep-0394/):
>  in preparation for an eventual change in the default version of Python, Python 2 only scripts should either be updated to be source compatible with Python 3 or else to use python2 in the shebang line.